components/Collections/examples/CollectionsALaCarteExample.m

```diff
@@ -61,6 +61,12 @@ - (void)viewDidLoad {
                                                  collectionViewLayout:self.collectionViewLayout];
   self.collectionView = _customCollectionView;
 
+#if defined(__IPHONE_11_0) && (__IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0)
+  if (@available(iOS 11.0, *)) {
+    self.collectionView.contentInsetAdjustmentBehavior = UIScrollViewContentInsetAdjustmentAlways;
```

Comment:
I think it is better to have it on every class and not override this behavior in MDCCollectionViewController, because UICollectionViewController does not.

Reply:
Do we need both checks? Xcode 8 and earlier do not have `__IPHONE_11_0` defined in the SDK, and the max allowed is `__IPHONE_10_3` in Xcode 8.3.2.

Reply:
:)